assets/js/comments.js

```diff
@@ -10,6 +10,10 @@ String.prototype.supplant = function (o) {
     });
 };
 
+function updateReplyLinkHtml(contentHtml) {
+    return contentHtml.replace(/target="_blank" href="\/comment_viewer\?[^"]*"/g, 'href="javascript:void(0)"');
+};
+
```

Comment:
I'm not super found of that tbh :/
Is there a cleaner way, using the DOM APIs?

Reply:
I've updated it to use DOM APIs instead